### PR TITLE
Fix balance in fees section

### PR DESCRIPTION
**Pull request by KarishmaBothara**
Have fixed the fees balance on extrinsic page..
Will need to search more places for balance and fix it..
Have changed the form of output for rpc calls from toHuman to toString as toHuman will internally call formatBalance with unit and withSi.
Will discuss with the team on the output..
Screen Shot 2020-07-16 at 3 06 07 PM

### CODE REVIEW

File: packages/react-api/src/Api.tsx [hunk 1]
```diff
@@ -67,7 +67,8 @@ async function loadOnReady (api: ApiPromise): Promise<State> {
   // TODO: query from genericAsset.assetRegistry
   // 1) find spending Asset ID
   // 2) find spending Asset ID decimals and symbol
-  const tokenSymbol = 'CPAY';
+  // const tokenSymbol = 'CPAY';
```
Comment (alexsednz) [inline packages/react-api/src/Api.tsx:70]: Can we remove this line?

File: packages/react-signer/src/Modal.tsx [hunk 2]
```diff
@@ -598,7 +598,7 @@ class Signer extends React.PureComponent<Props, State> {
       console.log('submitRpc: result ::', format(result));
 
       return {
-        result,
+        result: result,
```
Comment (alexsednz) [inline packages/react-signer/src/Modal.tsx:601]: Is it necessary?

Comment (KarishmaBothara) [reply packages/react-signer/src/Modal.tsx]: no

**Comment by KarishmaBothara** [comment]
As discussed with @holygits have only kept these changes for display of fees balance. Have reverted changes related to storage and rpc as we plan to create a new ticket with details on how output of those calls will appear.

**Comment by holygits** [comment]
I think transfer modal still needs some change
Screen Shot 2020-07-16 at 4 07 32 PM
Screen Shot 2020-07-16 at 4 07 25 PM

File: packages/react-components/src/util/getFormatedBalance.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { formatBalance } from '@polkadot/util';
+
+
```
Comment (holygits) [inline packages/react-components/src/util/getFormatedBalance.ts:3]: Add comment.
Some spec tests would be good, it's not really clear how `value` will be transformed 
I think it's truncating the value so input `0.1234` is becoming `0.1230`

File: packages/react-components/test/getFormatedBalance.spec.ts [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+      expect(getFormattedBalance(value, showUnit)).toEqual('0.0001');
+    });
+    it('formats 1234 (decimals=4, with no unit)', () => {
+      const value = new BN(1234);
+      const showUnit = false;
+      formatBalance.setDefaults({
+        decimals: 4,
+        unit: 'XYZ'
+      });
+      expect(getFormattedBalance(value, showUnit)).toEqual('0.0123');
```
Comment (holygits) [inline packages/react-components/test/getFormatedBalance.spec.ts:76]: shouldn't `1234` -> `0.1234`?

File: packages/react-components/test/getFormatedBalance.spec.ts [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+      expect(getFormattedBalance(TESTVAL3, showUnit)).toEqual('0.123 XYZ');
+    });
+    it('formats 1 (decimals=2, with no unit)', () => {
+      const value = new BN(1);
+      const showUnit = false;
+      formatBalance.setDefaults({
+        decimals: 2,
+        unit: 'XYZ'
+      });
+      expect(getFormattedBalance(value, showUnit)).toEqual('0.10');
```
Comment (holygits) [inline packages/react-components/test/getFormatedBalance.spec.ts:58]: `12` => `0.12`

**Pull request merged**